DatadogSessionReplay/SRSnapshotTests/SRFixtures/Sources/SRFixtures/ViewControllers/SwiftUIView.swift

```diff
@@ -0,0 +1,66 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import SwiftUI
+
+@available(iOS 13.0, *)
```

Comment:
question/ do we have a target only for snapshot testing? I'm wondering why the View is set for a low version while being used only for snapshots. 
Looking at the env var, it seems pointing to 17.5 (`DEFAULT_SR_SNAPSHOT_TESTS_OS := 17.5`)

Reply:
Totally! I added these UI elements bit by bit to test, and didn't clean up at the end. I removed all these checks to only keep a single one for the whole class.